docs/examples/asyncagi/example.php

```diff
@@ -37,8 +37,7 @@
     implode(
```

Comment:
This whole ini_set can probably be removed, as long as composers `vendor/autoload.php` is included. That is actually the whole idea of composer :smile: